bin/varnishtest/tests/proxyclose.vtc

```diff
@@ -0,0 +1,31 @@
+varnishtest "Pass buffering with early close"
```

Comment:
please keep the vtc nomenclature.

Reply:
FTR, this applies to all the VTCs
That said, I think we could have more descriptive vtc names, but if so, only after a deliberate decision has been made.

Reply:
Probably tests in the `v*.vtc` category.